file-formats/smbc/type/zif_aff_smbc_v1.intf.abap

```diff
@@ -118,68 +118,84 @@ INTERFACE zif_aff_smbc_v1
     "! <p class="shorttext synchronized">Object Page Settings</p>
     BEGIN OF ty_object_page,
       "! <p class="shorttext synchronized">Entity Set</p>
-      entity_set              TYPE string,
+      entity_set                     TYPE string,
       "! <p class="shorttext synchronized">Section Layout</p>
       "! Toggle between onepage concept and tabs.
       "! $values {@link zif_aff_smbc_v1.data:co_section_layout}
       "! $default {@link zif_aff_smbc_v1.data:co_section_layout.page}
-      section_layout          TYPE string,
+      section_layout                 TYPE string,
       "! <p class="shorttext synchronized">Variant Management</p>
       "! $values {@link zif_aff_smbc_v1.data:co_page_variant_management}
       "! $default {@link zif_aff_smbc_v1.data:co_page_variant_management.none}
-      variant_management      TYPE string,
+      variant_management             TYPE string,
       "! <p class="shorttext synchronized">Editable Header Content</p>
       "! If true, the header content is changeable in edit mode
-      editable_header_content TYPE abap_bool,
+      editable_header_content        TYPE abap_bool,
+      "! <p class="shorttext synchronized">Show Transport Selection Strip</p>
+      "! If true, the selected transport request and transport request select action are displayed in a message strip.
+      show_transport_selection_strip TYPE abap_bool,
     END OF ty_object_page,
     "! <p class="shorttext synchronized">Table Settings</p>
     BEGIN OF ty_table_setting,
       "! <p class="shorttext synchronized">Entity Set</p>
-      entity_set             TYPE string,
-      table_type             TYPE ty_table_type,
+      entity_set                    TYPE string,
+      table_type                    TYPE ty_table_type,
       "! <p class="shorttext synchronized">Selection Mode</p>
       "! $values {@link zif_aff_smbc_v1.data:co_selection_mode}
-      "! $default {@link zif_aff_smbc_v1.data:co_selection_mode.auto}
-      selection_mode         TYPE string,
+      "! $default {@link zif_aff_smbc_v1.data:co_selection_mode.multi}
```

Comment:
I think this is an incompatible change! If you really want to change the default, I think you need to introduce a new format version.

Is changing the default really necessary?

Reply:
@schneidermic0 
The "Auto" setting has been deprecated, see https://sapui5.hana.ondemand.com/sdk/#/topic/116b5d82e8c545e2a56e1b51b8b0a9bd
The new default "Multi" for practical purposes corresponds to the deprecated "Auto". You can compare the description of the "Auto" mode following this link: https://ui5.sap.com/1.96.37/#/topic/116b5d82e8c545e2a56e1b51b8b0a9bd

We are aware that during de-serialization of existing objects the selectionMode can now automatically switch from "Auto" to "Multi"

Reply:
OK, thanks for the update. I understand changing the default from "auto" to "multi" is (semantically) a compatible change.